Library/Formula/swatchbooker.rb

```diff
@@ -5,9 +5,9 @@ class Swatchbooker < Formula
   sha256 "c0c8bf038156337f1eebdf6f7c99f5b7a8e8f9a332cd625c57269ddc8ba18eb7"
 
   depends_on :python
-  depends_on "pillow" => [:python, "PIL"]
   depends_on "little-cms" => "with-python"
   depends_on "pyqt"
+  depends_on "homebrew/python/pillow"
```

Comment:
We don’t allow core formulae to depend on tap ones. If this is a mandatory requirement we must move this formula to the Python tap.